Formula/g/gcalcli.rb

```diff
@@ -20,43 +20,41 @@ class Gcalcli < Formula
   end
 
   depends_on "python-certifi"
-  depends_on "python-pyparsing"
```

Comment:
if we are going to remove `pyparsing` should we bump the revision?

Reply:
There's is plenty of stuff still using `python-pyparsing`, so I think we can wait.